share/completions/jhipster.fish

```diff
@@ -0,0 +1,37 @@
+# JHipster 4.9.0
+
+# Check if command already given
+function __fish_prog_needs_command
+    set cmd (commandline -opc)
+    echo $cmd
+    if [ (count $cmd) -eq 1 -a ]
```

Comment:
`-a` is for combining with another expression. Since you removed the other part it is doing nothing. It should go.